.github/workflows/test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Test spread
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt install -y xdelta3
+
+      - name: Setup go version
+        uses: actions/setup-go@v2
+        with:
+          # Using the lastest available go version for bionic
+          go-version: 1.13.8
+
+      - name: Set environment
+        run: |
+          echo "GOHOME=${{ github.workspace }}" >> "$GITHUB_ENV"
+          echo "GOPATH=${{ github.workspace }}" >> "$GITHUB_ENV"
+
+      - name: Checkout project
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/snapcore/spread
+
+      - name: Build spread
+        run: |
+          cd ${{ github.workspace }}/src/github.com/snapcore/spread
+          go get ./cmd/spread
+          
+      - name: run tests
+        env:
+          SPREAD_GOOGLE_KEY: ${{ secrets.SPREAD_GOOGLE_KEY }}
+        run: |
+          cd ${{ github.workspace }}/src/github.com/snapcore/spread
+          mkdir -p bin
+          cp -f "$GOHOME/bin/spread" ./bin/spread
+          "$GOHOME/bin/spread" google:ubuntu-20.04-64:tests/
```

Comment:
Maybe less specific here? Just ` "$GOHOME/bin/spread" google:` ? So that if we add compat with 22.04 we don't need to change this part?